frontend/.env.example

```
@@ -109,4 +109,6 @@ RATE_LIMIT_DISABLED=0
 
 GROQ_API_KEY=
 
-NEXT_PUBLIC_WEB3FORMS_KEY=
\ No newline at end of file
+NEXT_PUBLIC_WEB3FORMS_KEY=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The NEXT_PUBLIC_WEB3FORMS_KEY key is duplicated

(DuplicatedKey)